RULES.md

```diff
@@ -0,0 +1,150 @@
+- Add between a few (1-4) tags.
+- **Prioritize existing tags**.
+- As a general guideline, add the country from which the recipe originates if the recipe is representative of said country, using the adjective form (eg. *mexican*, *italian*, etc).
+- Tag the main ingredient if it's something slightly special.
+
+##### List of special, categorical tags to use if relevant
+
+- `basic`: for basic recipes that aren't meant to be standalone but are supposed
+  to be incorporated in another recipe.
+- `breakfast`
+- `cheesefare`: for recipes that include dairy but no milk.
+- `dessert`
+- `drink`
+- `quick`: for recipes that can be cooked in under ~20 minutes.
+- `side`: side dishes such as mash, fries, etc.
+- `snack`
+- `spread`
+- `strict-fast`: for recipes that contain no meat or dairy.
+- `vegan`
+- `vegetarian`
+
+#### Images
+
+- Images are stored in `data/pix`.
+- Each recipe can have a title image at the top and perhaps several instructional images as absolutely necessary.
+- Do not add stock images you found on the internet.
+Take a good picture yourself of the actual dish you prepared.
+- If you see a bad or substandard image, you may submit a better one.
+- Images should be in `.webp` format and with as small a file size as possible.
+If you submit an image for say, `chicken-parmesan.md`, it should be added as `pix/chicken-parmesan.webp`.
+- If you would like to add additional directional images, they should be numbered with two digits like `pix/chicken-parmesan-00.webp`, `pix/chicken-parmesan-01.webp`, etc.
+
+### Bugmanism is banned
+
+Vegan or vegetarian recipes are fine, but specific bugman food imitating authentic food is disallowed.
+
+You could always substitute any ingredient for any other ingredient, or follow any instruction any way you like whilst you are preparing the dish, for example to swap meat out for a substitute.
+However, do not propose these personal changes to [based.cooking](https://based.cooking).
+
+#### Examples
+
+- Never use a meat substitute instead of real meat.
+Non-meat pretending to be meat, like soy in the shape of sausages, or mycoprotein meat substitutes, are banned.
+- Tofu is only allowed when used in an authentic dish, and not as a meat substitute.
+
+### No soyboys
+
+No soydev code, Javascript, WebAssembly, or other bloat is permitted.
+
+Soybeans and soy-based food are acceptable.
+
+### No meme-tier sweet or unhealthy dishes
+
+Only substantial dishes are allowed, no hyper-sugary desserts or junk food.
+
+### Treat your audiance as if they are intelligent
+
+- Do not include ubiquitous ingredients like salt and pepper in the ingredients list.
+- Keep your instructions clear and concise.
+- Do not waffle on or elaborate in unnecessary ways.
+
+### No life stories
+
+Keep the introduction short.
+Nobody cares about the 1000-page lore and backstory of your recipe's cinematic universe.
+
+### No smoothies
+
+Milkshakes are fine, but Luke doesn't like smoothies for some reason.
```

Comment:
I think it is more just to do with the fact that smoothies are mostly things like milk plus fruit in a blender. So all smoothies end up being pretty much the same and self explanatory from the title.

Milkshakes (from my logic above) would also be rejected though

Reply:
@LukeSmithxyz we need clarification. Is is that these recipes end up being too similar?

I noticed you removed smoothies but there is still a milkshake on the site.

Reply:
Maybe specific smoothies should just be added to a general "smoothie.md" page with all the variations of smoothie that can possibly exist in this universe. (split into categories like milk smoothies, cider smoothies, etc.)